slices/gpgv.yaml

```diff
@@ -0,0 +1,23 @@
+package: gpgv
+
+essential:
+  - gpgv_copyright
+
+slices:
+  bins:
+    essential:
+      - base-files_tmp
+      - coreutils_mktemp
```

Comment:
why? there is no mention of `mktemp` in the whole dep. even `strings /usr/bin/gpgv | grep mktemp` yields nothing.